### ISSUE TITLE
docs: Issue in page Administration Guide

### ISSUE THREAD

**Issue by dorlaor** [scylladb/scylladb#12606]
I would like to report an issue in page https://docs.scylladb.com/branch-5.1/operating-scylla/admin

### Problem
Document timeout configuration

###  Suggest a fix

Like this:
[centos@ip-10-0-0-59 scylla]$ egrep '^[^#]+timeout_in_ms:' /etc/scylla/scylla.yaml
read_request_timeout_in_ms: 5000
write_request_timeout_in_ms: 2000
cas_contention_timeout_in_ms: 1000
range_request_timeout_in_ms: 10000
request_timeout_in_ms: 10000

**Comment by tzach** [comment]
related: https://github.com/scylladb/scylladb/issues/11585 and https://github.com/scylladb/scylladb/issues/8424

@annastuchlik This is yet another example of the lack of configuration docs.
The info is all in the code; we need to extract it.

**Comment by annastuchlik** [comment]
I agree, and it's added to the list of improvements we're going to work on and release with 5.3 and 5.4.
Initially, I tried to address this very issue by manually adding the information, but I resigned, as it would be hardly maintainable (for example, if a default changes).

**Comment by annastuchlik** [comment]
The timeout configuration is now documented along with other parameters, on the Configuration Parameters page.
The content of this page is automatically pulled from the config.cc file.
Examples:
https://docs.scylladb.com/manual/stable/reference/configuration-parameters.html#confval-write_request_timeout_in_ms
https://docs.scylladb.com/manual/stable/reference/configuration-parameters.html#confval-read_request_timeout_in_ms 

Closing.
